sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/misc.scala

```diff
@@ -57,9 +57,10 @@ case class Md5(child: Expression) extends UnaryExpression with ImplicitCastInput
  * the hash length is not one of the permitted values, the return value is NULL.
  */
 @ExpressionDescription(
-  usage = "_FUNC_(input, bitLength) - Returns a checksum of SHA-2 family as a hex string of the " +
-    "input. SHA-224, SHA-256, SHA-384, and SHA-512 are supported. Bit length of 0 is equivalent " +
-    "to 256",
+  usage =
+    """_FUNC_(input, bitLength) - Returns a checksum of SHA-2 family as a hex string of the input.
+      SHA-224, SHA-256, SHA-384, and SHA-512 are supported. Bit length of 0 is equivalent to 256."""
+  ,
```

Comment:
@yhuai that is the weird thing. One concatenation is fine, more than one isn't. Seems like a bug in scala :s...

I think it is better to be consistent and to only allow raw strings in case of multi-line ```usage/extended``` fields.